scripts/utils/recommendation_system.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
+from pandas import DataFrame
+from scripts.clusters.helpers import draw_graph, get_clusters, \
+get_clusters_count, get_sum_of_square_errors, cluster_predict
+
+class RecommendationSystem:
+
+    def __init__(
+        self, 
+        dataset: DataFrame) -> None:
+        self.__dataset = dataset
+
+
+    def build_system(self, x_cols:list):
+        features = self.__dataset[x_cols]
+        kmeans_kwargs = {
+            'init': 'random',
+            'n_init': 10,
+            'max_iter': 500,
+            'random_state': 42,
+        }
+
+        max_kernels = 30
+        sse = get_sum_of_square_errors(features, max_kernels, kmeans_kwargs)
+        draw_graph(x_range=range(1, max_kernels+1), y=sse, 
```

Comment:
I don't think the user would expect this method to draw some graph when running, maybe make it optional via additional parameter or create a separate method to draw it